src/incubator/f5-bigip-ingress/README.md

```diff
@@ -20,4 +34,22 @@ Or
 helm install -f path/to/custom-values.yaml charts/src/incubator/f5-bigip-ingress/
 ```
 
+## Primary Chart parameters:
+
+Parameter | Description | Default
+----------|-------------|--------
+ingress.annotations.virtual-server.f5.com/ip | IP accepting traffic on the BIG-IP | **Required** no default
+ingress.annotations.virtual-server.f5.com/partition | BIG-IP partition of the Controller | **Required** no default
+ingress.namespace | Kubernetes/OpenShift namespace for the ingress | Optional
+spec | Backend(s) and associated hosts and paths | See [examples](https://github.com/F5Networks/charts/tree/master/example_values/f5-bigip-ingress) 
+
+### Additional Optional parameters as annotations
+
+The annotations listed under the `ingress.annotations` parameter are consumed as an array and any of the [documentend annotations for the k8s-bigip-ctlr](http://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/latest/#supported-ingress-annotations) may be used.
```

Comment:
"The annotations listed" --> "The Annotations listed"
"documented annotations" --> "documented Annotations"

Reply:
Done.